# HG changeset patch
# User Daniel J Sebald <daniel.sebald@ieee.org>
# Date 1476517890 18000
#      Sat Oct 15 02:51:30 2016 -0500
# Branch stable
# Node ID 37b7b86f62f2ea18232928cffa5ebf5afc7d944b
# Parent  7adc5bbd840a7d880fda2412171782dc994494eb
Fix legend.m typo in demo example 23 (bug #49341).

* legend.m: For demo 23 illustrating "legend off", make the legend strings
  length 11 and replace double quote " (string) with single (quote transpose).
  Place a one second pause between showing the legend and turning off the
  legend, for dramatic effect.

diff --git a/scripts/plot/appearance/legend.m b/scripts/plot/appearance/legend.m
--- a/scripts/plot/appearance/legend.m
+++ b/scripts/plot/appearance/legend.m
@@ -1414,17 +1414,18 @@ endfunction
 
 %!demo
 %! clf;
 %! x = 0:10;
 %! plot (x, rand (11));
 %! xlabel ("Indices");
 %! ylabel ("Random Values");
 %! title ('Legend "off" deletes the legend');
-%! legend (cellstr (num2str ((1:10)")), "location", "northeastoutside");
+%! legend (cellstr (num2str ((0:10)')), "location", "northeastoutside");
+%! pause (1);
 %! legend off;
 %! axis ([0, 10, 0 1]);
 
 %!demo
 %! clf;
 %! x = (1:5)';
 %! subplot (2,2,1);
 %!  plot (x, rand (numel (x)));
